MovieApp/Screens/Home/Views/ListCell/ListItemCell/ListItemCellCV.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ListItemCellCV.swift
+//  MovieApp
+//
+//  Created by MinhDev on 10/12/21.
+//
+
+import UIKit
+import Model
+
+class ListItemCellCV: UICollectionViewCell {
```

Comment:
```suggestion
class ListItemCell: UICollectionViewCell {
```

@minh5896 : What is CV ? Please define name follow with suffix *Cell*

Reply:
fix complêt